application/templates/application/index.html

```diff
@@ -8,10 +8,16 @@
     <div class="all-app-root">
         <div class="container-fluid">
             <div class="row">
-                <div class="col-md-12 left-pane">
-                    {% block app_list %}
-                        {% include 'application/application-list.html' %}
+                <div class="col-md-12 left-pane" style="overflow-y: scroll;">
```

Comment:
Using "overflow" feature for this div causes the applications page to have multiple scrollbars. Usually, we try to avoid this, but occasionally it serves well for some UI/UXs. Please do comment if you feel we should keep it this way!

Reply:
I couldn't find another way of doing this as the form was overflowing. Can you suggest any solution for this?

Reply:
So, I have attached a few files on slack for you. If we remove the overflow from this div, the outer body div's overflow method catches the form and that way we can achieve this